Signal/test/ViewControllers/ConversationViewItemTest.m

```diff
@@ -0,0 +1,347 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "ConversationViewItem.h"
+#import <MobileCoreServices/MobileCoreServices.h>
+#import <SignalServiceKit/SecurityUtils.h>
+#import <SignalServiceKit/TSAttachmentStream.h>
+#import <SignalServiceKit/TSOutgoingMessage.h>
+#import <XCTest/XCTest.h>
+
+@interface ConversationViewItem (Testing)
+
+- (SEL)copyActionSelector;
+- (SEL)saveActionSelector;
+- (SEL)shareActionSelector;
+- (SEL)deleteActionSelector;
+- (SEL)metadataActionSelector;
+
+@end
+
+@interface ConversationViewItemTest : XCTestCase
+
+@property (nonatomic, readonly) NSData *fakeAudioData;
+@property (nonatomic, readonly) NSData *fakeImageData;
+
+@end
+
+@implementation ConversationViewItemTest
+
+- (NSData *)fakeAudioData
```

Comment:
Now that we're including real assets in the test suite, can these `fake*Data` methods go away?

Reply:
Yup!